### PR TITLE
remoing the unused file-profile services

### LINKED ISSUE #48: Request to update getFileProfileFilePath,putFileProfileFilePath with getFileProfileFileName,putFileProfileFileName in the controller File-profile.js
**Issue Discription:** The Oam services getFileProfileFileName and putFileProfileFileName  are getting failing with 500 Internal server Error Since the file-path is changed to file -name 

Solution: update getFileProfileFilePath,putFileProfileFilePath with getFileProfileFileName,putFileProfileFileName under the controller File-profile.js and in the service FileProfileServices.js

### PR DESCRIPTION
**Pull request by ManasaBM1**
Fixes #48

### CODE REVIEW

**Comment by ManasaBM1** [comment]
Reviewers is @PrathibaJee

File: server/controllers/FileProfile.js [hunk 1]
```diff
@@ -59,35 +59,9 @@ module.exports.getFileProfileOperation = async function getFileProfileOperation(
...
-    })
-    .catch(function (response) {
-      let sentResp = responseBuilder.buildResponse(res, undefined, response);
-      responseCode = sentResp.code;
-    });
-  oamLogService.recordOamRequest(req.url, req.body, responseCode, req.headers.authorization, req.method);
-};
-
-module.exports.putFileProfileFilePath = async function putFileProfileFilePath(req, res, next, body, uuid) {
+module.exports.putFileProfileFileName = async function putFileProfileFileName(req, res, next, body, uuid) {
```
Comment (PrathibaJee) [inline server/controllers/FileProfile.js:62]: PUT filename is creating the following problem , 
```
{
  "code": 400,
  "message": "request.body['file-profile-1-0:file-name'] should match pattern \"^([A-Za-z0-9-_,\\s]{3,})\\.([A-Za-z]{3})$\"",
  "errors": [
    {
      "path": ".body['file-profile-1-0:file-name']",
      "message": "should match pattern \"^([A-Za-z0-9-_,\\s]{3,})\\.([A-Za-z]{3})$\"",
      "errorCode": "pattern.openapi.validation"
    }
  ]
}
```
for the input 
{
  "file-profile-1-0:file-name": "application-data.json"
}

Please have a look into the YAML file and kindly look for the possible rootcause.

**Review by PrathibaJee — CHANGES_REQUESTED** [review]
Could you please kindly look into the provided review comment. 
Many thanks.

File: server/controllers/FileProfile.js (hunk 1 above)
Comment (ManasaBM1) [reply server/controllers/FileProfile.js:62]: This commit does not using a updated yaml file so it is giving this error

**Pull request merged**